### PR TITLE
New README

### PR DESCRIPTION
**Pull request by philipkiely-baseten**
A quick refresh to the README to bring it more up-to-date with Truss 0.10 plans

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -6,7 +6,7 @@ We use GitHub features for project management on Truss:
 
 * For bugs and feature requests, file an issue.
 * For changes and updates, create a pull request.
-* To view and comment on the roadmap, check the projects tab.
+* To view and comment on the roadmap, [check the projects tab](https://github.com/basetenlabs/truss/projects?query=is%3Aopen).
```
Comment (bolasim) [inline CONTRIBUTING.md:9]: please use this link instead: 
https://github.com/orgs/basetenlabs/projects/3

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -58,17 +58,4 @@ To release a new version of the library.
...
-
-[Baseten Labs, Inc](http://baseten.co)
-* Phil Howes
-* Alex Gillmor
-* Pankaj Gupta
-* Philip Kiely
-* Nish Singaraju
-* Abu Qadar
-* and users like you!
+Contributions to documentation are very welcome! Simply edit the appropriate markdown files in the `docs/` folder and make a pull request. For larger changes, tutorials, or any questions please contact [philip.kiely@baseten.co](mailto:philip.kiely@baseten.co)
```
Comment (bolasim) [inline CONTRIBUTING.md:61]: can we please use team@trussml.com?

File: README.md [hunk 3]
```diff
@@ -1,145 +1,92 @@
...
-
-Get started with the [end-to-end tutorial](https://truss.baseten.co/e2e).
-
-## What can I do with Truss?
-
-If you've ever tried to get a model out of a Jupyter notebook, Truss is for you.
+## Why Truss?
 
-Truss exposes just the right amount of complexity around things like Docker and APIs without you really having to think about them. Here are some of the things Truss does:
+* **Write once, deploy anywhere:** A model packaged with Truss can be deployed on supported providers without changing the underlying server implementation.
```
Comment (bolasim) [inline README.md:10]: it's without changing anything from the user perspective? I feel like we don't wanna say server implementation here/

File: README.md [hunk 4]
```diff
@@ -1,145 +1,92 @@
...
 
-* 🏎 Turns your Python model into a microservice with a production-ready API endpoint, no need for Flask or Django.
-* 🎚 For most popular frameworks, includes automatic model serialization and deserialization.
-* 🛍 Freezes dependencies via Docker to make your training environment portable.
-* 🕰 Enables rapid iteration with local development that matches your production environment.
-* 🗃 Encourages shipping parsing and even business logic alongside your model with integrated pre- and post-processing functions.
-* 🤖 Supports running predictions on GPUs. (Currently limited to certain hardware, more coming soon)
-* 🙉 Bundles secret management to securely give your model access to API keys.
+See Trusses for popular models including [Falcon 40B](https://github.com/basetenlabs/falcon-40b-truss), [WizardLM](https://github.com/basetenlabs/wizardlm-truss), [Stable Diffusion](https://github.com/basetenlabs/stable-diffusion-truss), [Whisper](https://github.com/basetenlabs/whisper-truss), and [dozens more examples](examples/).
```
Comment (bolasim) [inline README.md:14]: Would a bullet list with emojis be good here?

File: README.md [hunk 5]
```diff
@@ -1,145 +1,92 @@
...
+        return self._model(model_input)
 ```
 
-Then, as long as the container is running, you can invoke the model as an API as follows:
+There are two functions to implement:
 
-```
-curl -X POST http://127.0.0.1:8080/v1/models/model:predict -d '[[0, 0, 0, 0]]'
-```
+* `load()` runs once when the model is spun up and is responsible for setting `self._model`
```
Comment (bolasim) [inline README.md:60]: setting -> initializing

File: README.md [hunk 6]
```diff
@@ -1,145 +1,92 @@
...
-The Truss we generated above in the quickstart sample has a good example of a typical Truss config:
+In `./text-classification/config.yaml`, find the line `requirements`. Replace the empty list with:
 
 ```yaml
-model_framework: sklearn
-model_metadata:
-  model_binary_dir: model
-  supports_predict_proba: true
-python_version: py39
 requirements:
```
Comment (bolasim) [inline README.md:70]: would be cool if we can show a diff view here

**Review by bolasim — COMMENTED** [review]
LGTM. Just some suggestions. Will wait on approving til the Remote PR is merged.

File: README.md (hunk 6 above)
Comment (philipkiely-baseten) [reply README.md]: Learned some new markdown syntax. Only issue is it makes it super annoying to copy and paste, so I'm going to revert to a non-diff version.

Screenshot 2023-07-05 at 5 28 55 PM

**Comment by philipkiely-baseten** [comment]
@bolasim everything should be addressed from your review, feel free to merge whenever Truss push is ready!

File: README.md (hunk 6 above)
Comment (bolasim) [reply README.md:77]: hmm. I feel the diff is cleaner even if the copy and paste doesn't work. You can probably actually disable copy and paste (at least you could with mkdocs)

File: README.md [hunk 7]
```diff
@@ -1,145 +1,99 @@
...
 [![PyPI version](https://badge.fury.io/py/truss.svg)](https://badge.fury.io/py/truss)
 [![ci_status](https://github.com/basetenlabs/truss/actions/workflows/main.yml/badge.svg)](https://github.com/basetenlabs/truss/actions/workflows/main.yml)
 
-Meet Truss, a seamless bridge from model development to model delivery. Truss presents an open-source standard for packaging models built in any framework for sharing and deployment in any environment, local or production.
-
-Get started with the [end-to-end tutorial](https://truss.baseten.co/e2e).
+## Why Truss?
 
-## What can I do with Truss?
+* **Write, deploy, invoke:** Package and test model code, weights, and dependencies with a model server that behaves the same in development and production.
```
Comment (bolasim) [inline README.md:10]: What happened to write once, run anywhere?

Comment (philipkiely-baseten) [reply README.md:10]: Oops, was playing with other tag lines and forgot to put it back. Will put it back.

**Review by bolasim — APPROVED** [review]

**Pull request merged**